BFS_Connected_Components/Q1_number_of_transformations_bfs.py

```diff
@@ -0,0 +1,37 @@
+import collections
+
+# BFS
+def get_min_operations_bfs(start_num, target_num, additive_nums, multiplicative_nums):
+
+    def get_neighbors(num):
+        neighbors = [num * multiplicative_num for multiplicative_num in multiplicative_nums]
+        neighbors.extend([num + additive_num for additive_num in additive_nums])
+        return neighbors
+
+
+    q = collections.deque([(start_num, 0)])
+    seen = set()
+    seen.add(start_num)
+
+    while q:
+        new_num, depth = q.popleft()
```

Comment:
`depth` is not a clear name for what this variable represents. Something like `num_operations_so_far` would be better.